virtual_ecosystem/models/animal/animal_cohorts.py

```diff
@@ -1012,28 +1004,284 @@ def delta_mass_herbivory(
 
         return total_consumed_cnp
 
+    def calculate_consumed_mass_detritivory(
+        self,
+        litter_pools: list[Resource],
+        target_pool: Resource,
+    ) -> float:
+        """Return kg wet mass a cohort removes from one litter pool over Δt.
+
+        The search/handling formulation is identical to herbivory; only the resource
+          list and pool class differ.
+
+        Args:
+            litter_pools: All litter pools available in the territory.  Used
+                to compute the denominator of handling time.
+            target_pool: The specific pool from which biomass is sought.
+
+        Returns:
+            Wet mass (kg) of litter requested from ``target_pool`` during the
```

Comment:
I don't think this is really a wet mass as the litter model only tracks the carbon, nitrogen and phosphorus contents